### PR TITLE
CCD-3807 hmc-cft-hearing-service-int

### PR DESCRIPTION
**Pull request by nathan-clark**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/hmc/hmc-cft-hearing-service-int/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: hmc-cft-hearing-service-int
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/hmc/cft-hearing-service:prod-6ff8e63-20221103114640 #{"$imagepolicy": "flux-system:hmc-cft-hearing-service"}
```
Comment (thomast1906) [inline k8s/namespaces/hmc/hmc-cft-hearing-service-int/demo.yaml:8]: want to update to image policy you are adding ?

Comment (nathan-clark) [reply k8s/namespaces/hmc/hmc-cft-hearing-service-int/demo.yaml:8]: we can change when need be

**Review by thomast1906 — APPROVED** [review]
providing an int environment is needed

**Review by timja — CHANGES_REQUESTED** [review]
all new apps / jobs must use flux v2

**Comment by nathan-clark** [comment]
> 
ah, I assumed there was a migration script in the works

**Comment by nathan-clark** [comment]
closing will redo in v2

**Pull request closed without merging**